src/components/views/elements/AccessibleButton.tsx

```diff
@@ -86,6 +87,14 @@ type Props<T extends keyof JSX.IntrinsicElements> = DynamicHtmlElementProps<T> &
      * Event handler for button activation. Should be implemented exactly like a normal `onClick` handler.
      */
     onClick: ((e: ButtonEvent) => void | Promise<void>) | null;
+    /**
+     * The tooltip to show on hover or focus.
+     */
+    title?: string;
+    /**
+     * The caption to show within a tooltip as above, only valid when used in conjunction with `title`.
```

Comment:
What's a caption, in this sense? What does it mean to "show within a tooltip"?

Also, comma splice. s/,/./

Reply:
I added more informations